microsoft-edge/index.yml

```diff
@@ -411,7 +411,7 @@ conceptualContent:
               url: ./devtools-guide-chromium/json-viewer/json-viewer.md
               itemType: how-to-guide
 
-            - text: Sign up for the Ad Selection API
+            - text: Sign up for the Ad Selection API  # todo: moved to "Experimental web platform features" toc bucket, delete?
```

Comment:
Delete "Sign up for the Ad Selection API" from "Edge-specific web development tips" card of hub page?
Article moved to "Experimental web platform features" toc bucket.